exterminate.lua

```diff
@@ -109,6 +95,8 @@ function killUnit(unit, method)
     elseif method == killMethod.VAPORIZE then
         vaporizeUnit(unit)
     elseif method == killMethod.DISINTEGRATE then
+        -- Teleport them to world height. Out of sight, out of mind.
+        dfhack.units.teleport(unit, xyz2pos(unit.pos.x, unit.pos.y, df.global.world.map.z_count-1))
```

Comment:
does it work if we instead mark them as hidden (in ambush)? it seems like teleporting has more things that can go wrong (e.g. what if your castle in the sky happens to have a kinder care center at that spot?)

Reply:
The idea is to prevent smoke clouds as well, maybe teleport them to Hell?